src/heatmaps.ts

```diff
@@ -160,4 +190,14 @@ export class Heatmaps {
 
         this.buffer[url].push(properties)
     }
+
+    private flush(): void {
+        if (!this.buffer || isEmptyObject(this.buffer)) {
+            return
+        }
+
+        this.instance.capture('$$heatmap', {
```

Comment:
Why does heatmaps use the double $?

Reply:
i believe it makes it not chargeable - it's like our marker for internal events instead of posthog events - trying to figure that out currently cos I can drop it completely in pipeline once we've extracted the data